product-mini/platforms/nuttx/test_wasm.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2020 XiaoMi Corporation.  All rights reserved.
```

Comment:
This file is copied from WAMR project, could you please change the copyright?

Reply:
Done